instance/api/instance.py

```diff
@@ -117,3 +118,21 @@ def app_servers(self, request, pk):
         Get this Instance's entire list of AppServers
         """
         return Response(InstanceAppServerSerializer(self.get_object(), context={'request': request}).data)
+
+    @detail_route(methods=['post'])
+    def update_attrs(self, request, pk):
```

Comment:
I'm not sure we want the ability to update any attribute of an instance reference here. Perhaps a more focused `set_notes` handler?

Reply:
I was thinking about using this endpoint for future instance updates, but I can limit the scope for now.